metadata-db-app/src/migrations/048_setup_variables_table.clj

```diff
@@ -0,0 +1,58 @@
+(ns migrations.048-setup-variables-table
+  (:require
+   [config.mdb-migrate-helper :as h]))
+
+(def ^:private variables-column-sql
+  "id NUMBER,
+  concept_id VARCHAR(255) NOT NULL,
+  native_id VARCHAR(1030) NOT NULL,
+  metadata BLOB NOT NULL,
+  format VARCHAR(255) NOT NULL,
+  revision_id INTEGER DEFAULT 1 NOT NULL,
+  revision_date TIMESTAMP WITH TIME ZONE DEFAULT SYSTIMESTAMP NOT NULL,
+  created_at TIMESTAMP WITH TIME ZONE DEFAULT SYSTIMESTAMP NOT NULL,
+  deleted INTEGER DEFAULT 0 NOT NULL,
+  user_id VARCHAR(30),
+  variable_name VARCHAR(20),
```

Comment:
This length is from the [Variable schema](https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/browse/DRAFT/umm-var-json-schema.json) that Simon has posted.